docs/basic-features/script.md

```diff
@@ -174,6 +174,26 @@ export default function Home() {
 }
 ```
 
+Sometimes it is helpful to catch when a script fails to load. These events can be caught with the `onError` property:
```

Comment:
```suggestion
Sometimes it is helpful to catch when a script fails to load. These errors can be handled with the `onError` property:
```

Reply:
I'm not sure I agree with your suggested changes. Why do you think "error" is better than "event"?

Reply:
The returned parameter is an error object

Reply:
fair.